bin/deploy

```diff
@@ -45,13 +42,14 @@ usage() {
   # note that heredocs are prefixed with tab chars
   fail "$(cat <<-EOF
 
-		Usage: $0 [-2] [-3] [-b] [-h]
+		Usage: $0 [-2] [-3] [-b] [-h] [-r]
 
 		Specific options:
-		    -b          use bleeding edge conda packages
-		    -3          use Python 3 (default)
-		    -2          use Python 2 (no longer supported -- fatal error)
-		    -h          show this message
+	    -b          use bleeding edge conda packages
+	    -r REF      git ref in scipipe_conda_env (hash, branch, tag)
+	    -3          use Python 3 (default)
+	    -2          use Python 2 (no longer supported -- fatal error)
+	    -h          show this message
```

Comment:
Minor comment, but why did the tabbing change here? It makes the diff bigger than it really is (which was adding a `-r` option.

Reply:
I don't know... I did not change the original formatting, so I just fixed the tab problem as KT pointed out.

Reply:
I think your editor may be reformatting this?  You need to make this a one-line diff somehow, with that one line matching the formatting of the others when this is executed.